docs/concepts/linq.rst

```diff
@@ -0,0 +1,268 @@
+LINQ
+====
+
+What is it?
+-----------
+
+LINQ provides language-level querying capabilities and `higher-order function <https://en.wikipedia.org/wiki/Higher-order_function>`_ API to C# and VB (note that F# already has this capability by nature of being a functional programming language).  It is used either via native query syntax,
```

Comment:
> Can you add C# and VB examples to that wikipedia page? There is an F# sample there.

Yep, will do.

> Hm, weren't higher-order functions already partially supported via delegates, and especially anonmyous delegates?

Yes; `Action` and `Func` are examples of this (and they're used in the API).  I'm calling it out because it's one of the two ways you write LINQ.

> Also, the above sentence reads like F# already had LINQ before C# and VB.

Changing this to be its own sentence.